### PR TITLE
Sitemap, Index page img alt text and title change, link to buttons

### PR DESCRIPTION
**Pull request by andrewjcasal**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/index_new.html [hunk 1]
```diff
@@ -82,27 +82,27 @@ <h1 class="fade">Automate your team's development environments</h1>
...
-      <a href="#" id="greenButton">Learn more</a>
+      <a href="{{site.url}}/verticals/training" id="greenButton">Learn more</a>
     </li><li>
       <div>Software trial automation</div>
       <p>Your potential users can evaluate your software with the click of a button</p>
-      <a href="#" id="redButton">Learn more</a>
+      <a href="{{site.url}}/verticals/trial" id="redButton">Learn more</a>
     </li><li>
       <div>Open source</div>
       <p>Install Koding on premisis for both personal use and enterprise solutions</p>
```
Comment (sinan) [inline docs/index_new.html]: premises*

File: docs/index_new.html [hunk 2]
```diff
@@ -82,27 +82,27 @@ <h1 class="fade">Automate your team's development environments</h1>
     <div class="u-headline">Closing copy blah blah blah</div>
```
Comment (sinan) [inline docs/index_new.html]: proper copy pls

File: docs/robots.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+User-agent: *
+Sitemap: http://www.koding.com/sitemap.xml
```
Comment (sinan) [inline docs/robots.txt]: 👍

**Review by sinan — CHANGES_REQUESTED** [review]

File: docs/index_new.html (hunk 2 above)
Comment (andrewjcasal) [inline docs/index_new.html]: There was no proper copy given to me for this. That being said, I will flag down Devrim for copy.

**Comment by sinan** [comment]
@andrewjcasal please send the sitemap.xml with another PR. closing this since we don't need the other changes anymore.

**Pull request closed without merging**